Accordion.js

```diff
@@ -0,0 +1,496 @@
+/** @module deliteful/Accordion */
+define(["dcl/dcl",
+	"decor/sniff",
+	"requirejs-dplugins/Promise!",
+	"delite/register",
+	"delite/KeyNav",
+	"requirejs-dplugins/jquery!attributes/classes",
+	"delite/DisplayContainer",
+	"./ToggleButton",
+	"delite/theme!./Accordion/themes/{{theme}}/Accordion.css"
+], function (dcl, has, Promise, register, KeyNav, $, DisplayContainer, ToggleButton) {
+
+	function setVisibility(node, val) {
+		node.style.display = val ? "" : "none";
+	}
+
+	var animationEndEvent = (function () {
```

Comment:
I don't remember if we've already discussed this but this 'hack' is becoming some sort of pattern which many animated widgets are likely to need. We might want to consider putting it somewhere it can be shared.